src/include/stir/TimeFrameDefinitions.h

```diff
@@ -107,6 +107,8 @@ class TimeFrameDefinitions
 
     //! Set time frame
     void set_time_frame(const int frame_num, const double start, const double end);
+    
+    bool operator == (const TimeFrameDefinitions &t) const { return this->frame_times==t.frame_times;}
```

Comment:
probably best to do a float/safe comparison here. It's going to trip us up otherwise. Best to put it in the .cxx then

Reply:
you mean like a comparison of each element of the vector?

Reply:
yes, with some accounting for rounding error.

Reply:
> 
> 
> yes, with some accounting for rounding error.

I was thinking something like that:

`bool TimeFrameDefinitions::operator == (const TimeFrameDefinitions &t) const {
    bool is_identical=0;
    
    for (int frame=0;frame<frame_times.size(); frame++){
        
        is_identical=((frame_times.at(frame).first == t.frame_times.at(frame).first) &&
                      (frame_times.at(frame).second == t.frame_times.at(frame).second));
    }
    return is_identical;
}

however I guess this does not allow  for rounding errors. Peraphs I should compare the differences with and arbitrary small number, like 10^-6?